src/registrar/tests/test_reports.py

```diff
@@ -606,6 +619,61 @@ def test_write_body_with_date_filter_pulls_domains_in_range(self):
 
             self.assertEqual(csv_content, expected_content)
 
+    def test_export_domains_to_writer_domain_managers(self):
+        """Test that export_domains_to_writer returns the
+        expected domain managers"""
+        with less_console_noise():
+            # Create a CSV file in memory
+            csv_file = StringIO()
+            writer = csv.writer(csv_file)
+            # Define columns, sort fields, and filter condition
+
+            columns = [
+                "Domain name",
+                "Status",
+                "Expiration date",
+                "Domain type",
+                "Agency",
+                "Organization name",
+                "City",
+                "State",
+                "AO",
+                "AO email",
+                "Security contact email",
+            ]
+            sort_fields = ["domain__name"]
+            filter_condition = {
+                "domain__state__in": [
+                    Domain.State.READY,
+                    Domain.State.DNS_NEEDED,
+                    Domain.State.ON_HOLD,
+                ],
+            }
+            self.maxDiff = None
+            # Call the export functions
+            write_body(writer, columns, sort_fields, filter_condition, True, True)
```

Comment:
Great call out - do you mean for every time the function is called (ie in `csv_export.py` and `test_reports.py`)? I do agree it could be useful for readability, but this function is referenced a decent few times so I hoped that ppl would reference the main function to see which one was which -- open to either way!

Reply:
Ideally yes but if its a lot of effort to implement, no worries on that. Its a minor thing